src/test/java/com/softdevsix/argos/DemoApplicationTests.java

```diff
@@ -0,0 +1,20 @@
+package com.softdevsix.argos;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
+class DemoApplicationTests {
```

Comment:
Please rename this application to the application name "DemoApplication" It is generated automatically